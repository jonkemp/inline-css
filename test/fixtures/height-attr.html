<html>
<head>
    <style>
        p, td, img {
            height: 200px;
        }
    </style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <td>
                tall
            </td>
        </tr>
    </table>
    <img src="" alt="tall">
</body>
</html>
