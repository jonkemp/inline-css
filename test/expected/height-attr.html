<html>
<head>
    
</head>
<body>
    <p style="height: 200px;">none</p>
    <table>
        <tr>
            <td style="height: 200px;" height="200">
                tall
            </td>
        </tr>
    </table>
    <img src="" alt="tall" style="height: 200px;" height="200">
</body>
</html>
